rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -258,6 +258,7 @@ struct spu_channel_4_t
 
 		sync_var.atomic_op([&out_result, &out_value, &out_count, last_value](sync_var_t& data)
 		{
+			// TODO:: Syphurith: I don't know what you tried to express, exactly.
```

Comment:
@Nekotekina Oh well. Then should i classify it with `()`, before remove the comment?